lang/cs/Org.Apache.REEF.All/Org.Apache.REEF.All.csproj

```diff
@@ -26,7 +26,7 @@ under the License.
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <RestorePackages>true</RestorePackages>
-    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..</SolutionDir>
+    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
```

Comment:
We usually use it in the way like $(SolutionDir)\ where slash is followed. Adding one more slash will end up two slashes.